Sources/Sentry/Public/SentryOptions.h

```diff
@@ -230,6 +230,13 @@ NS_SWIFT_NAME(Options)
  */
 @property (nullable, nonatomic, weak) id<NSURLSessionDelegate> urlSessionDelegate;
 
+/**
+ * Controls if the `tracestate` header is attached to envelopes and HTTP client integrations.
+ *
+ * Note: this is an experimental API and will be removed without notice.
+ */
+@property (nonatomic, readonly, assign) BOOL enableTraceSampling;
```

Comment:
on Java, we use `@ApiStatus.Experimental`, so totally in favor of making it experimental by a prefix, Python is also a Map, but they are not statically typed languages, I feel that for iOS makes more sense to prefix them as well instead of having a `Map<String, Bool>`, in the future when making it stable, we can always add the `enableTraceSampling` flag and `experimentalEnableTraceSampling` forwards the call to `enableTraceSampling` for compatibility, and if we have to remove it, well, its an `experimental` feature.

Reply:
The `options.experiment` wouldn't be a map, but instead, it would be a class. For keeping it simple, let's just go with the prefix. We can anyway change this later.